resources/Microsoft.Windows.Developer/Microsoft.Windows.Developer.psd1

```diff
@@ -1,46 +1,44 @@
-@{
```

Comment:
Do we know why github erreounsly beleives that I edited all of the Microsoft.Windows.Developer files when they were only surgical edits?